distribution/lib/Standard/Test/0.0.0-dev/src/Test_Reporter.enso

```diff
@@ -193,11 +194,12 @@ print_progress current_progress total_count status_text =
     truncated_line = if line.length <= progress_width then line else
         line.take (progress_width - 3) + '...'
 
-    Java_System.out.print '\r'
-    Java_System.out.print (' ' * progress_width)
-    Java_System.out.print '\r'
-    Java_System.out.print truncated_line
-    Java_System.out.print '\r'
+    ps = Java_System.out : PrintStream
```

Comment:
To invoke `print` we have to include `java.io.PrintStream` in the list of classes needed for _reflective access_ - see https://github.com/enso-org/enso/pull/9997#discussion_r1606033800 - the other option is to create `Text_Utils.print(PrintStream out, String text)` and call such a dedicated static method instead.

Reply:
Reported as 
- #10028